### PR TITLE
People 174 add description for skill categories

### PR DESCRIPTION
**Pull request by DariuszMusielak**
# Ticket
https://netguru.atlassian.net/browse/PEOPLE-174

# Description
- Added explanation for skill categories

### Before merge checklist
- [ ] CircleCI is passing
- [ ] Proper labels are set

### CODE REVIEW

File: app/assets/stylesheets/components/btn-modal.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.btn-info {
+  &--modal {
+    position: fixed;
```
Comment (trefnis) [inline app/assets/stylesheets/components/btn-modal.scss]: Probably you could use standard utility: `@include fixed(bottom 30px right 25px);` here.

Comment (DariuszMusielak) [inline app/assets/stylesheets/components/btn-modal.scss]: Yeaj, forgot about thi. I have to do it in that way:
`@include position(fixed, null 25px 30px null);`

File: app/views/shared/_requested_changes_modal.html.haml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            %b.text-info created
+            %span - when requested change is new, and ready to review.
+          %li
+            %b.text-success accepted
+            %span - when requested change is accepted.
+          %li
+            %b.text-danger declined
+            %span - when requested change is declined.
+      .modal-footer
+        %button.btn.btn-primary{ data: { dismiss: 'modal' }, type: 'button' } Got it!
```
Comment (netguru-hound) [inline app/views/shared/_requested_changes_modal.html.haml]: Line is too long. [85/80]

File: app/views/shared/_requested_changes_modal.html.haml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  %i.glyphicon.glyphicon-question-sign
+#requestedChangeModal.modal.fade{ aria: { labelledby: 'requestedChangeModalLabel' }, role: 'dialog', tabindex: '-1' }
+  .modal-dialog{ role: 'document' }
+    .modal-content
+      .modal-header
+        %button.close{ aria: { label: 'Close' }, data: { dismiss: 'modal' }, type: 'button' }
+          %span{ aria: { hidden: 'true' } } ×
+        %h4#categoryModalLabel.modal-title Requested changes
+      .modal-body
+        %span Table represents all requested changes. We have two types of requested change:
```
Comment (netguru-hound) [inline app/views/shared/_requested_changes_modal.html.haml]: Line is too long. [92/80]

File: app/views/shared/_requested_changes_modal.html.haml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+.btn.btn-info.btn-info--modal.btn-xs{ data: { target: '#requestedChangeModal', toggle: 'modal' }, type: 'button' }
+  %i.glyphicon.glyphicon-question-sign
+#requestedChangeModal.modal.fade{ aria: { labelledby: 'requestedChangeModalLabel' }, role: 'dialog', tabindex: '-1' }
+  .modal-dialog{ role: 'document' }
+    .modal-content
+      .modal-header
+        %button.close{ aria: { label: 'Close' }, data: { dismiss: 'modal' }, type: 'button' }
+          %span{ aria: { hidden: 'true' } } ×
+        %h4#categoryModalLabel.modal-title Requested changes
```
Comment (netguru-hound) [inline app/views/shared/_requested_changes_modal.html.haml]: Classes should be listed before IDs (.modal-title should precede #categoryModalLabel)

File: app/views/shared/_requested_changes_modal.html.haml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+.btn.btn-info.btn-info--modal.btn-xs{ data: { target: '#requestedChangeModal', toggle: 'modal' }, type: 'button' }
+  %i.glyphicon.glyphicon-question-sign
+#requestedChangeModal.modal.fade{ aria: { labelledby: 'requestedChangeModalLabel' }, role: 'dialog', tabindex: '-1' }
+  .modal-dialog{ role: 'document' }
+    .modal-content
+      .modal-header
+        %button.close{ aria: { label: 'Close' }, data: { dismiss: 'modal' }, type: 'button' }
```
Comment (netguru-hound) [inline app/views/shared/_requested_changes_modal.html.haml]: Line is too long. [93/80]

File: app/views/shared/_requested_changes_modal.html.haml [hunk 6]
```diff
@@ -0,0 +1,29 @@
+.btn.btn-info.btn-info--modal.btn-xs{ data: { target: '#requestedChangeModal', toggle: 'modal' }, type: 'button' }
+  %i.glyphicon.glyphicon-question-sign
+#requestedChangeModal.modal.fade{ aria: { labelledby: 'requestedChangeModalLabel' }, role: 'dialog', tabindex: '-1' }
```
Comment (netguru-hound) [inline app/views/shared/_requested_changes_modal.html.haml]: Line is too long. [117/80]<br>Classes should be listed before IDs (.modal should precede #requestedChangeModal)

File: app/views/shared/_requested_changes_modal.html.haml [hunk 7]
```diff
@@ -0,0 +1,29 @@
+.btn.btn-info.btn-info--modal.btn-xs{ data: { target: '#requestedChangeModal', toggle: 'modal' }, type: 'button' }
```
Comment (netguru-hound) [inline app/views/shared/_requested_changes_modal.html.haml]: Line is too long. [114/80]

File: app/assets/stylesheets/components/btn-modal.scss [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+    line-height: 25px;
+    color: $color-white;
+    text-shadow: none;
+    border: none;
+    border-radius: 15px;
+    background-color: $color-light-blue;
+    box-shadow: 0 3px 9px $color-black-transparent;
+    z-index: 100;
+
+    &:active, &:hover {
```
Comment (netguru-hound) [inline app/assets/stylesheets/components/btn-modal.scss]: Each selector in a comma sequence should be on its own single line

File: app/assets/stylesheets/components/btn-modal.scss [hunk 9]
```diff
@@ -0,0 +1,19 @@
+.btn-info {
+  &--modal {
+    @include position(fixed, null 25px 30px null);
+    padding: 0;
+    font-size: 30px;
+    line-height: 25px;
+    color: $color-white;
+    text-shadow: none;
+    border: none;
```
Comment (netguru-hound) [inline app/assets/stylesheets/components/btn-modal.scss]: `border: 0` is preferred over `border: none`

File: app/assets/stylesheets/components/btn-modal.scss [hunk 10]
```diff
@@ -0,0 +1,19 @@
+.btn-info {
+  &--modal {
+    @include position(fixed, null 25px 30px null);
+    padding: 0;
```
Comment (netguru-hound) [inline app/assets/stylesheets/components/btn-modal.scss]: Properties should be ordered background-color, border, border-radius, box-shadow, color, font-size, line-height, padding, text-shadow, z-index

File: app/assets/stylesheets/bootstrap_overrides/_modal.scss [hunk 11]
```diff
@@ -0,0 +1,6 @@
+.modal-content {
+  display: block;
+  margin-top: 40%;
+  border-radius: 0;
+  border: none;
```
Comment (netguru-hound) [inline app/assets/stylesheets/bootstrap_overrides/_modal.scss]: `border: 0` is preferred over `border: none`

File: app/assets/stylesheets/bootstrap_overrides/_modal.scss [hunk 12]
```diff
@@ -0,0 +1,6 @@
+.modal-content {
+  display: block;
```
Comment (netguru-hound) [inline app/assets/stylesheets/bootstrap_overrides/_modal.scss]: Properties should be ordered border, border-radius, display, margin-top

File: app/views/shared/_notifications_modal.html.haml [hunk 13]
```diff
@@ -0,0 +1,20 @@
...
+        %span On this page you can see list of notifications from 30 days. We have two different types of notifications:
+        %ul
+          %li
+            %b.text-info Skill updated
+            %span - you receive it when skill you have rated have been edited.
+          %li
+            %b.text-success Skill created
+            %span - you receive it when new skill have been added.
+      .modal-footer
+        %button.btn.btn-primary{ data: { dismiss: 'modal' }, type: 'button' } Got it!
```
Comment (netguru-hound) [inline app/views/shared/_notifications_modal.html.haml]: Line is too long. [85/80]

File: app/views/shared/_notifications_modal.html.haml [hunk 14]
```diff
@@ -0,0 +1,20 @@
...
+  %i.glyphicon.glyphicon-question-sign
+#categoryModal.modal.fade{ aria: { labelledby: 'categoryModalLabel' }, role: 'dialog', tabindex: '-1' }
+  .modal-dialog{ role: 'document' }
+    .modal-content
+      .modal-header
+        %button.close{ aria: { label: 'Close' }, data: { dismiss: 'modal' }, type: 'button' }
+          %span{ aria: { hidden: 'true' } } ×
+        %h4#categoryModalLabel.modal-title Notifications
+      .modal-body
+        %span On this page you can see list of notifications from 30 days. We have two different types of notifications:
```
Comment (netguru-hound) [inline app/views/shared/_notifications_modal.html.haml]: Line is too long. [120/80]

File: app/views/shared/_notifications_modal.html.haml [hunk 15]
```diff
@@ -0,0 +1,20 @@
+.btn.btn-info.btn-info--modal.btn-xs{ data: { target: '#categoryModal', toggle: 'modal' }, type: 'button' }
+  %i.glyphicon.glyphicon-question-sign
+#categoryModal.modal.fade{ aria: { labelledby: 'categoryModalLabel' }, role: 'dialog', tabindex: '-1' }
+  .modal-dialog{ role: 'document' }
+    .modal-content
+      .modal-header
+        %button.close{ aria: { label: 'Close' }, data: { dismiss: 'modal' }, type: 'button' }
+          %span{ aria: { hidden: 'true' } } ×
+        %h4#categoryModalLabel.modal-title Notifications
```
Comment (netguru-hound) [inline app/views/shared/_notifications_modal.html.haml]: Classes should be listed before IDs (.modal-title should precede #categoryModalLabel)

File: app/views/shared/_notifications_modal.html.haml [hunk 16]
```diff
@@ -0,0 +1,20 @@
+.btn.btn-info.btn-info--modal.btn-xs{ data: { target: '#categoryModal', toggle: 'modal' }, type: 'button' }
+  %i.glyphicon.glyphicon-question-sign
+#categoryModal.modal.fade{ aria: { labelledby: 'categoryModalLabel' }, role: 'dialog', tabindex: '-1' }
+  .modal-dialog{ role: 'document' }
+    .modal-content
+      .modal-header
+        %button.close{ aria: { label: 'Close' }, data: { dismiss: 'modal' }, type: 'button' }
```
Comment (netguru-hound) [inline app/views/shared/_notifications_modal.html.haml]: Line is too long. [93/80]

File: app/views/shared/_notifications_modal.html.haml [hunk 17]
```diff
@@ -0,0 +1,20 @@
+.btn.btn-info.btn-info--modal.btn-xs{ data: { target: '#categoryModal', toggle: 'modal' }, type: 'button' }
+  %i.glyphicon.glyphicon-question-sign
+#categoryModal.modal.fade{ aria: { labelledby: 'categoryModalLabel' }, role: 'dialog', tabindex: '-1' }
```
Comment (netguru-hound) [inline app/views/shared/_notifications_modal.html.haml]: Line is too long. [103/80]<br>Classes should be listed before IDs (.modal should precede #categoryModal)

File: app/views/shared/_notifications_modal.html.haml [hunk 18]
```diff
@@ -0,0 +1,20 @@
+.btn.btn-info.btn-info--modal.btn-xs{ data: { target: '#categoryModal', toggle: 'modal' }, type: 'button' }
```
Comment (netguru-hound) [inline app/views/shared/_notifications_modal.html.haml]: Line is too long. [107/80]

File: app/views/shared/_categories_modal.html.haml [hunk 19]
```diff
@@ -0,0 +1,15 @@
...
+      .modal-header
+        %button.close{ aria: { label: 'Close' }, data: { dismiss: 'modal' }, type: 'button' }
+          %span{ aria: { hidden: 'true' } } ×
+        %h4#categoryModalLabel.modal-title Skill categories
+      .modal-body
+        %span Skill categories represents different sections at Netguru. Each section collets skills which we want to have as a company. We have one more special category.
+        %b Shared
+        %span which contains similar skills for couple categories.
+      .modal-footer
+        %button.btn.btn-primary{ data: { dismiss: 'modal' }, type: 'button' } Got it!
```
Comment (netguru-hound) [inline app/views/shared/_categories_modal.html.haml]: Line is too long. [85/80]

File: app/views/shared/_categories_modal.html.haml [hunk 20]
```diff
@@ -0,0 +1,15 @@
...
+  %i.glyphicon.glyphicon-question-sign
+#categoryModal.modal.fade{ aria: { labelledby: 'categoryModalLabel' }, role: 'dialog', tabindex: '-1' }
+  .modal-dialog{ role: 'document' }
+    .modal-content
+      .modal-header
+        %button.close{ aria: { label: 'Close' }, data: { dismiss: 'modal' }, type: 'button' }
+          %span{ aria: { hidden: 'true' } } ×
+        %h4#categoryModalLabel.modal-title Skill categories
+      .modal-body
+        %span Skill categories represents different sections at Netguru. Each section collets skills which we want to have as a company. We have one more special category.
```
Comment (netguru-hound) [inline app/views/shared/_categories_modal.html.haml]: Line is too long. [171/80]

File: app/views/shared/_categories_modal.html.haml [hunk 21]
```diff
@@ -0,0 +1,15 @@
+.btn.btn-info.btn-info--modal.btn-xs{ data: { target: '#categoryModal', toggle: 'modal' }, type: 'button' }
+  %i.glyphicon.glyphicon-question-sign
+#categoryModal.modal.fade{ aria: { labelledby: 'categoryModalLabel' }, role: 'dialog', tabindex: '-1' }
+  .modal-dialog{ role: 'document' }
+    .modal-content
+      .modal-header
+        %button.close{ aria: { label: 'Close' }, data: { dismiss: 'modal' }, type: 'button' }
+          %span{ aria: { hidden: 'true' } } ×
+        %h4#categoryModalLabel.modal-title Skill categories
```
Comment (netguru-hound) [inline app/views/shared/_categories_modal.html.haml]: Classes should be listed before IDs (.modal-title should precede #categoryModalLabel)

File: app/views/shared/_categories_modal.html.haml [hunk 22]
```diff
@@ -0,0 +1,15 @@
+.btn.btn-info.btn-info--modal.btn-xs{ data: { target: '#categoryModal', toggle: 'modal' }, type: 'button' }
+  %i.glyphicon.glyphicon-question-sign
+#categoryModal.modal.fade{ aria: { labelledby: 'categoryModalLabel' }, role: 'dialog', tabindex: '-1' }
+  .modal-dialog{ role: 'document' }
+    .modal-content
+      .modal-header
+        %button.close{ aria: { label: 'Close' }, data: { dismiss: 'modal' }, type: 'button' }
```
Comment (netguru-hound) [inline app/views/shared/_categories_modal.html.haml]: Line is too long. [93/80]

File: app/views/shared/_categories_modal.html.haml [hunk 23]
```diff
@@ -0,0 +1,15 @@
+.btn.btn-info.btn-info--modal.btn-xs{ data: { target: '#categoryModal', toggle: 'modal' }, type: 'button' }
+  %i.glyphicon.glyphicon-question-sign
+#categoryModal.modal.fade{ aria: { labelledby: 'categoryModalLabel' }, role: 'dialog', tabindex: '-1' }
```
Comment (netguru-hound) [inline app/views/shared/_categories_modal.html.haml]: Line is too long. [103/80]<br>Classes should be listed before IDs (.modal should precede #categoryModal)

File: app/views/shared/_categories_modal.html.haml [hunk 24]
```diff
@@ -0,0 +1,15 @@
+.btn.btn-info.btn-info--modal.btn-xs{ data: { target: '#categoryModal', toggle: 'modal' }, type: 'button' }
```
Comment (netguru-hound) [inline app/views/shared/_categories_modal.html.haml]: Line is too long. [107/80]

**Pull request merged**